commercetools-models/src/main/java/io/sphere/sdk/shoppinglists/ShoppingList.java

```diff
@@ -76,6 +74,11 @@
     @Nullable
     String getAnonymousId();
 
+    @HasUpdateAction
+    @Nullable
+    @IgnoreInQueryModel
```

Comment:
why did you flagged it to be ignored in the query model?

Reply:
because without this annotation it gives me the error "java: cannot find symbol" in the querys, querybuilders, queryImpls, draftbuilders and so on classes